src/components/molecules/ResourceFilter/ResourceFilter.tsx

```diff
@@ -88,27 +90,27 @@ const ResourceFilter = () => {
   }, [resourceMap]);
 
   const resetFilters = () => {
-    setName(undefined);
+    setName('');
     setKind(ALL_OPTIONS);
     setNamespace(ALL_OPTIONS);
     setLabels({});
     setAnnotations({});
   };
 
   const updateKind = (selectedKind: string) => {
-    if (selectedKind === ALL_OPTIONS) {
-      setKind(undefined);
-    } else {
-      setKind(selectedKind);
-    }
+    // if (selectedKind === ALL_OPTIONS) {
```

Comment:
Wanted to ask why is this implemented in this way? What is the reason?

Reply:
It was needed to show the string `<all>` in the `Select` when the value is `undefined`